scripts/mysql_backup/install.sh

```
@@ -0,0 +1,206 @@
+
+# GitHub Actions와 수동 실행이 겹쳐도 하나의 설치 트랜잭션만 진행합니다.
+exec 200>"$INSTALL_LOCK_FILE"
+flock 200
+
+install -d -m 755 -o root -g root "$INSTALL_LIB_DIR" "$INSTALL_BIN_DIR" "$CONFIG_DIR"
+install -d -m 700 -o root -g root \
+  /mnt/mysql-data/mysql-backup \
+  /mnt/mysql-data/mysql-backup/staging \
+  /mnt/mysql-data/mysql-backup/state
+
+readonly TRANSACTION_DIR="$(mktemp -d /tmp/mysql-backup-install-transaction.XXXXXX)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)